app/views/seeds/index.rss.haml

```diff
@@ -16,8 +16,9 @@
             <p>GMO? #{seed.gmo}</p>
             <p>Heirloom? #{seed.heirloom}</p>
           - if seed.tradable?
-            :escaped
-              <p>Will trade #{seed.tradable_to} from #{seed.owner.location ? seed.owner.location : 'unknown location'}</p>
+            %p
```

Comment:
Let's remove all `:escaped` from our code base. I'll do this today

side note: our spammer is likely watching these conversations.